stock_inventory_non_stocked/tests/test_check_inventory_responsible.py

```diff
@@ -0,0 +1,38 @@
+# Copyright 2024 Ivan Perez <iperez@coninpe.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo.tests import common
+
+
+class TestCheckInventoryResponsible(common.TransactionCase):
+    def setUp(self):
```

Comment:
Please use `classmethod`instead.

Moreover, I can encourage you to inherit from `BaseCommon`class to improve performances: https://github.com/odoo/odoo/blob/16.0/odoo/addons/base/tests/common.py#L19